lib/commons/text/is-human-interpretable.js

```diff
@@ -13,28 +13,37 @@ import sanitize from './sanitize';
 function isHumanInterpretable(str) {
   /**
    * Steps:
-   * 1) Check for single character edge cases
+   * 1) Trim string to avoid whitespace
+   * 2) Check for symbolic text character edge cases
    * 		a) handle if character is alphanumeric & within the given icon mapping
-   * 					eg: x (close), i (info)
+   * 					eg: x (dismiss), i (info)
    *
-   * 3) handle unicode from astral  (non bilingual multi plane) unicode, emoji & punctuations
+   * 3) handle unicode from astral (non bilingual multi plane) unicode, emoji & punctuations
    * 					eg: Windings font
    * 					eg: '💪'
    * 					eg: I saw a shooting 💫
    * 					eg: ? (help), > (next arrow), < (back arrow), need help ?
    */
 
+  str = str.trim();
+
   if (!str.length) {
     return 0;
   }
 
   // Step 1
-  const alphaNumericIconMap = [
-    'x', // close
-    'i' // info
-  ];
+  const symbolicTextCharactersSet = new Set([
+    // dismiss
+    'x',
+    '×',
+    // wysiwyg
+    'i', // or, info
+    'b',
+    'aa', // toggle capitalization (aA)
```

Comment:
My recollection from last Thursday is that we decided that we wanted to make *all* single-character strings *except* digits 0-9 be treated as non-interpretable, rather than just extending the list of disallowed single-character cases.

I like your addition of aa/abc though, we should keep that in addition to the single-character thing.